components/stacks-network/src/orchestrator.rs

```diff
@@ -559,14 +597,108 @@ impl DevnetOrchestrator {
             }
         };
 
+        // Start stacks-signer-1
```

Comment:
Do we want booting the signers to be wrapped in `if use_nakamoto`?

Reply:
Good catch